### ISSUE TITLE
armor bar is broken

### ISSUE THREAD

**Issue by kingoscargames** [Wuzzy2/MineClone2-Bugs#249]
https://github.com/Wuzzy2/MineClone2-Bugs/issues/243
also when wearing a full set of leather the armor bar is just as filled as when wearing diamond

**Comment by kingoscargames** [comment]
Stupid mobile touch screen

**Comment by Gerold55** [comment]
Is it possible to fix the survival inventory (creative mode) so that it's similar to when you are in survival mode?

**Comment by Wuzzy2** [comment]
No. The survival inventory in creative mode is already very close to MC, except for the additional buttons.

**Comment by Gerold55** [comment]
Ok
